pkg/network/ebpf/c/tracer.c

```diff
@@ -1027,13 +1027,11 @@ static __always_inline struct sock* sk_buff_sk(struct sk_buff *skb) {
     return sk;
 }
 
-SEC("tracepoint/net/net_dev_queue")
-int tracepoint__net__net_dev_queue(struct net_dev_queue_ctx* ctx) {
-    CHECK_BPF_PROGRAM_BYPASSED()
-    struct sk_buff* skb = ctx->skb;
-    if (!skb) {
-        return 0;
-    }
+struct bpf_raw_tracepoint_args {
```

Comment:
Is this being used anywhere?

Reply:
Yeah, I missed the rename elsewhere. Fixing it now

Reply:
It shouldn't be needed once moving to the `BPF_PROG` macro.